shared/src/business/useCases/pdf/sanitizePdfInteractor.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * sanitizePdfInteractor
+ *
+ * @param {object} applicationContext the application context
+ * @param {object} providers the providers object
+ * @param {string} providers.key the key of the document to validate
+ * @returns {object} errors (null if no errors)
+ */
+exports.sanitizePdfInteractor = async (applicationContext, { key }) => {
+  let { Body: pdfData } = await applicationContext
+    .getStorageClient()
+    .getObject({
+      Bucket: applicationContext.environment.documentsBucketName,
+      Key: key,
+    })
+    .promise();
+
+  const { PDFDocument } = await applicationContext.getPdfLib();
+
+  const pdfDoc = await PDFDocument.load(pdfData, { ignoreEncryption: true });
+  const form = pdfDoc.getForm();
+  const fieldCount = form.getFields().length;
+
+  if (fieldCount > 0) {
+    form.flatten();
+    const pdfBytes = await pdfDoc.save();
+
+    await applicationContext.getPersistenceGateway().saveDocumentFromLambda({
+      applicationContext,
+      document: pdfBytes,
+      key,
+      useTempBucket: false,
+    });
+  }
+
+  return true;
```

Comment:
Nit: Is this value used for anything?

Reply:
This was copied to behave similarly to validatePdfInteractor.  Essentially if everything succeeds, we return `true`, but otherwise an error may be thrown.  Perhaps it should come out of both? I'm open to that, too.

Reply:
I think I would prefer it to be removed if it's not being consumed anywhere!